src/components/inputs/TextField.js

```diff
@@ -436,51 +433,19 @@ export default class TextField extends BaseInput {
   }
 
   renderExpandableInput() {
-    const {value} = this.state;
-    const {
-      style, 
-      floatingPlaceholder, 
-      placeholder, 
-      hideUnderline, 
-      renderExpandableInput, 
-      rightIconSource,
-      testID
-    } = this.getThemeProps();
-    const typography = this.getTypography();
-    const {lineHeight, ...typographyStyle} = typography;
-
-    const color = this.getStateColor(this.props.color || this.extractColorValue());
-    const inputStyle = [
-      this.styles.input,
-      hideUnderline && this.styles.inputWithoutUnderline,
-      {...typographyStyle},
-      {minHeight: lineHeight + (Constants.isAndroid ? 6 : 0)},
-      color && {color},
-      style
-    ];
+    const {renderExpandableInput, rightIconSource} = this.getThemeProps();
 
     if (_.isFunction(renderExpandableInput)) {
       return renderExpandableInput(this.getThemeProps());
     }
 
-    const shouldShowPlaceholder = _.isEmpty(value) && !floatingPlaceholder;
-
     return (
       <TouchableOpacity
         style={this.styles.expandableInput}
         activeOpacity={1}
         onPress={() => !this.isDisabled() && this.toggleExpandableModal(true)}
-        testID={testID}
```

Comment:
I think the testID here was meant for things like detox and triggering the press of `TouchableOpacity`
let's add testID with a suffix like we usually do - `${testID}.expandable` or something like that

Reply:
Yes. But in this case, this is where you trigger the click, equivalent to the input when you render just a textInput, so they should be treated the same in my opinion.